src-vue/src/components/PlayButton.vue

```diff
@@ -114,10 +171,24 @@ export default defineComponent({
             />
         </el-option-group>
     </el-select>
+    <el-progress
+        v-if="install_or_update"
```

Comment:
I don't think we should use a `v-if` here, because it removes the progress bar from interface when there's no downloading/extraction process running, meaning the play button will move on progress bar (dis)appearance.
You should use the `v-show` property instead.

Reply:
Addressed in 0ef0c49de0e0fdd9dd6e4a0d36b6892ccb86b1be